distribution/src/main/gateway/bin/scripts/metrics.statsD.start.bat

```diff
@@ -0,0 +1,32 @@
+@REM
```

Comment:
Please rename this file to avoid the uppercase D - it's just likely to catch people out ( especially on Linux)